cmd/operator-sdk/alpha/cmd.go

```diff
@@ -26,7 +28,10 @@ func NewCmd() *cobra.Command {
 		Short: "Run an alpha subcommand",
 	}
 
-	cmd.AddCommand(olm.NewCmd())
-	cmd.AddCommand(kubebuilder.NewCmd())
+	cmd.AddCommand(
+		olm.NewCmd(),
+		kubebuilder.NewCmd(),
+		bundle.NewCmd(),
```

Comment:
We need to pretty quickly decide where to put the OLM integration subcommands in the CLI. I think `alpha` is _probably_ okay for this PR, but we'll need a follow-up on the heels of this to promote these OLM `alpha` subcommands.